lib/response-generator/index.js

```diff
@@ -74,8 +87,47 @@ class ResponseGenerator {
 		}
 	}
 
+	static generateByFaker(fakerString) {
+		// Check if faker string is a template string
+		if(fakerString.includes('{{') && fakerString.includes('}}'))
+			return faker.fake(fakerString);
+
+		const fakerRegex = /^(?<namespace>\w+)\.(?<method>\w+)(?:\((?<argsString>.*)\))?$/.exec(
+			fakerString
+		);
+		if(!fakerRegex) {
+			throw new Error(
+				'Faker extension method is not in the right format. Expecting <namespace>.<method>(<args>) format.'
+			);
+		}
+
+		const { namespace, method, argsString } = fakerRegex.groups;
+		if(!(namespace in faker)) {
+			throw new Error(
+				`Invalid faker namespace used. Must be one of ${Object.keys(faker).join(
+					','
+				)}`
+			);
+		}
+		if(!(method in faker[namespace])) {
+			throw new Error(
+				`Method '${method}' not found in faker namespace '${namespace}'. Must be one of ${Object.keys(
+					faker[namespace]
+				).join(',')}`
+			);
+		}
+
+		const args = argsString ? JSON.parse(`[${argsString}]`) : [];
+
+		return faker[namespace][method](...args);
+	}
+
 	static generateArray(schema) {
-		return [this.generate(schema.items)];
+		let count = Number(schema['x-count']);
+		if(Number.isNaN(count) || count < 1)
+			count = 1;
+
+		return [...new Array(count).keys()].map(() => this.generate(schema.items));
```

Comment:
Calling `.keys()` is not needed here.
```suggestion
		return [...new Array(count)].map(() => this.generate(schema.items));
```

Reply:
Yep, you are correct. I just blindly copied pasted some code from StackOverflow. It's not often I find myself having to create arrays of a certain size.